server.js

```diff
@@ -0,0 +1,99 @@
+var mongoose = require('mongoose');
+// Require the Express Module
+var express = require('express');
+// Create an Express App
+var app = express();
+// Require body-parser (to receive post data from clients)
+var bodyParser = require('body-parser');
+var db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  // we're connected!
+});
+// Integrate body-parser with our App
+app.use(bodyParser.urlencoded({ extended: true }));
+// Require path
+var path = require('path');
+// Setting our Static Folder Directory
+app.use(express.static(path.join(__dirname, './static')));
+// Setting our Views Folder Directory
+app.set('views', path.join(__dirname, './views'));
+// Setting our View Engine set to EJS
+app.set('view engine', 'ejs');
+
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost/mongoose_db');
+  var MongooseSchema = new mongoose.Schema({
+   name: String,
+   info: String,
+   comment: String
+  }, { versionKey: false });
+  mongoose.model('Mongoose', MongooseSchema); // We are setting this Schema in our Models as 'User'
+  var Mongoose = mongoose.model('Mongoose') // We are retrieving this Schema from our Models, named 'User'
+// Routes
+// Root Request
+// app.get('/', function(req, res) {
```

Comment:
Clean out commented code that's no longer needed when possible, it just makes our code base more manageable and readable.